.github/workflows/validate-theme-entry.yml

```diff
@@ -0,0 +1,269 @@
+                        addError(`Your theme screenshot is too small, ${recommendedSize}`);
+                    } else if (imageMeta.width !==512 || image.height !== 288) {
+                        addWarning(`Theme theme screenshot size is not optimal, ${recommendedSize}`);
+                    }
+                } catch (e) {
+                    addError('The theme screenshot cannot be found.');
+                }
+
+                // only validate releases if version is included
+                if (manifest.hasOwnProperty('version')) {
+                    try {
+                        let release = await github.rest.repos.getReleaseByTag({
+                            owner,
+                            repo,
+                            tag: manifest.version,
+                        });
+
+                        const assets = release.data.assets || [];
+                        if (!assets.find(p => p.name === 'theme.css')) {
+                            addError('Your latest Release is missing the `theme.css` file.');
+                        }
+                        if (!assets.find(p => p.name === 'manifest.json')) {
+                            addError('Your latest Release is missing the `manifest.json` file.');
+                        }
+                    } catch (e) { }
+
+                }
+
+                try {
+                    await github.rest.licenses.getForRepo({ owner, repo });
+                } catch (e) {
+                    addWarning('Your repository does not include a license. It is generally recommended for open-source projects to have a license. Go to <https://choosealicense.com/> to compare different open source licenses.');
+                }
+
+            })();
+
+
+            if (errors.length > 0 || warnings.length > 0) {
+                let message = [`#### Hello!\n`];
+                message.push(`**I found the following issues in your theme submission**\n`);
+                if (errors.length > 0) {
+                    message.push(`**Errors:**\n`);
+                    message = message.concat(errors);
+                    message.push(`\n---\n`);
+                }
+                if (warnings.length > 0) {
+                    message.push(`**Warnings:**\n`);
+                    message = message.concat(warnings);
+                    message.push(`\n---\n`);
+                }
+                message.push(`<sup>This check was done automatically. Do <b>NOT</b> open a new PR for re-validation. Instead, to trigger this check again, make a change to your PR and wait a few minutes, or close and re-open it.</sup>`);
+
+
+                await github.rest.issues.createComment({
+                    issue_number: context.issue.number,
+                    owner: context.repo.owner,
+                    repo: context.repo.repo,
+                    body: message
+                });
+                if (warnings.length > 0) {
```

Comment:
This should be errors.length not warnings.length, and it should be outside to match up with where it is in the plugin script.